src/tox_ansible/ansible/__init__.py

```diff
@@ -13,6 +13,8 @@
 from ..tox_molecule_case import ToxMoleculeCase
 from .scenario import Scenario
 
+LOCAL_CONFIG_FILE = ".config/molecule/config.yml"
```

Comment:
Can we make this a configurable value that would be pulled from the `tox.ini` file? I typically use a global config file, but mine lives at a different path across multiple repositories. Additionally, do we need to update the calls to `molecule` itself to pass a `-c {LOCAL_CONFIG_FILE}` command line option?

Here's an example of how I currently invoke my shared Molecule config file: https://github.com/oasis-roles/ansible_collection_osp/blob/master/tox.ini#L9-L10

Reply:
Sure I will do that!

Reply:
@greg-hellings 

Here it is. 
However, I need to work on the tests which don't work at the moment.

Reply:
last patchset fix only the pylint issues

Reply:
@ssbarnea @greg-hellings 

Greg, here is the last version addressing your comment with tests coverage.

It manages both situation:
1- global molecule configuration file at the project level
2- base configuration file(s) coming from the tox.ini file

If both are present, base configuration file(s) will take precedence over the global molecule configuration file according to how ansible-molecule is working.

About the question, do we need to update the calls to molecule itself to pass -c {LOCAL_CONFIG_FILE} command line option? the answer is no because ansible-molecule is looking for .config/molecule/config.yml and will load it by default [1]

[1] - https://github.com/ansible-community/molecule/blob/master/src/molecule/shell.py#L92